### PR TITLE
CA-Chart - 1.0.4 Release

### PR DESCRIPTION
**Pull request by gjtempleton**
Publish release of CA Chart version 1.0.3 to include changes from #3575

### CODE REVIEW

**Review by mwielgus — APPROVED** [review]
/lgtm
/approve

File: index.yaml [hunk 1]
```diff
@@ -1,6 +1,28 @@
...
+      name: mgoodness
+    - email: guyjtempleton@googlemail.com
+      name: gjtempleton
+    - email: scott.crooks@gmail.com
+      name: sc250024
+    name: cluster-autoscaler-chart
+    sources:
+    - https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler
+    urls:
+    - https://github.com/kubernetes/autoscaler/releases/download/cluster-autoscaler-chart-1.0.3/cluster-autoscaler-chart-1.0.4.tgz
```
Comment (michelzanini) [inline index.yaml:24]: Do we not have an error here? Should it not be `cluster-autoscaler-chart-1.0.4`?

```
https://github.com/kubernetes/autoscaler/releases/download/cluster-autoscaler-chart-1.0.3/cluster-autoscaler-chart-1.0.4.tgz
```

vs

```
https://github.com/kubernetes/autoscaler/releases/download/cluster-autoscaler-chart-1.0.4/cluster-autoscaler-chart-1.0.4.tgz
```

Comment (michelzanini) [reply index.yaml:24]: yes, this is wrong, the download failed = 

```
Error: failed to download "https://github.com/kubernetes/autoscaler/releases/download/cluster-autoscaler-chart-1.0.3/cluster-autoscaler-chart-1.0.4.tgz" (hint: running `helm repo update` may help)
```

Comment (michelzanini) [reply index.yaml:24]: I created this PR to fix it https://github.com/kubernetes/autoscaler/pull/3577

**Pull request merged**